samples/sensor/6dof_motion_drdy/src/main.c

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2024 TDK Invensense
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/sensor.h>
+#include <stdio.h>
+
+static struct sensor_trigger data_trigger;
+
+/* Flag set from IMU device irq handler */
+static volatile int irq_from_device;
+
+/*
+ * Get a device structure from a devicetree node from alias
+ * "6dof_motion_drdy0".
+ */
+static const struct device *get_6dof_motion_device(void)
+{
+	const struct device *const dev = DEVICE_DT_GET(DT_ALIAS(6dof_motion_drdy0));
+
+	if (dev == NULL) {
+		/* No such node, or the node does not have status "okay". */
+		printk("\nError: no device found.\n");
+		return NULL;
+	}
```

Comment:
this is impossible, build will fail if macro doesn't resolve

Reply:
Ok I removed it. I copied this from the bosh,bme280 example, I though it was needed.